applications/ray/kuberay-tpu-webhook/main.go

```diff
@@ -86,7 +86,7 @@ func getNumTPUHostsFromTopology(topology string, acceleratorType string) (int32,
 	for i := 0; i < len(topologyVals); i++ {
 		dim, err := strconv.Atoi(topologyVals[i])
 		if err != nil {
-			klog.Errorf("Invalid topology: %s", err)
+			klog.V(0).InfoS("getNumTPUHostsFromTopology", "RayCluster", namespace+"/"+clusterName, "Invalid topology", err)
```

Comment:
This should be ErrorS?

Reply:
Done, it ends up working the same since if the topology or accelerator are missing the webhook won't do anything (doesn't see it as multi-host), but logging it as an error seems like better practice.